crypto/asn1/x_algor.c

```diff
@@ -110,13 +110,18 @@ int X509_ALGOR_copy(X509_ALGOR *dest, const X509_ALGOR *src)
         if ((dest->algorithm = OBJ_dup(src->algorithm)) == NULL)
 	    return 0;
 
-    if (src->parameter)
+    if (src->parameter) {
```

Comment:
May as well fix up this code while you are here..

if (src->parameter != NULL)

Reply:
Updated.